lonestar/analytics/cpu/betweennesscentrality/AsyncStructs.h

```diff
@@ -400,26 +341,42 @@ struct BetweenessCentralityAsync {
   }
 };
 
-static const char* name = "Betweenness Centrality";
-static const char* desc = "Computes betwenness centrality in an unweighted "
-                          "graph";
-
-int main(int argc, char** argv) {
-  galois::SharedMemSys G;
-  LonestarStart(argc, argv, name, desc, nullptr, &inputFile);
-
-  galois::StatTimer totalTime("TimerTotal");
-  totalTime.start();
+void AsyncSanity(AsyncGraph& graph) {
+  galois::GReduceMax<float> accumMax;
```

Comment:
Going forward we are using snake_case for variable names and camelCase for function names. This might be too much for this jut mentioning here for future reference.